contributing-guides/translation-templates/subcommand-mention.md

```diff
@@ -124,7 +124,7 @@ Not translated yet.
 ---
 ### pt_BR
 
-Not translated yet.
+Alguns subcomandos como `exemplo comando` tem sua própia documentação de uso.
```

Comment:
Since this is a placeholder for an untranslated command, we keep it in English
```suggestion
Alguns subcomandos como `example command` tem sua própia documentação de uso.
```